core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/BootMonitors.scala

```diff
@@ -344,7 +346,8 @@ object BootMonitors extends LazyLogging {
                                             gcpBatchBackend: CromwellBackend,
                                             methodConfigResolver: MethodConfigResolver,
                                             bardService: BardService,
-                                            workspaceSettingRepository: WorkspaceSettingRepository
+                                            workspaceSettingRepository: WorkspaceSettingRepository,
+                                            useBatchAsDefaultBackend: Boolean
```

Comment:
Once the migration is done, I bet we could clean up more than just `useBatchAsDefaultBackend` from this parameter list.